runtime/gc_base/GCExtensions.cpp

```diff
@@ -252,3 +226,35 @@ MM_GCExtensions::updateIdentityHashDataForSaltIndex(UDATA index)
 {
 	getJavaVM()->identityHashData->hashSaltTable[index] = (U_32)convertValueToHash(getJavaVM(), getJavaVM()->identityHashData->hashSaltTable[index]);
 }
+
+void
+MM_GCExtensions::computeDefaultMaxHeap(MM_EnvironmentBase *env)
+{
+	int32_t rc = 0;
```

Comment:
The variables can be defined at first use since this is a cpp file.  Keeping variable life times short makes it easier to reason about the code.